scripts/depth/demo.py

```diff
@@ -0,0 +1,221 @@
+import os
+import argparse
+import time
+import PIL.Image as pil
+import numpy as np
+
+import mxnet as mx
+from mxnet.gluon.data.vision import transforms
+
+import gluoncv
+from gluoncv.model_zoo.monodepthv2.layers import disp_to_depth
+
+import matplotlib as mpl
+import matplotlib.cm as cm
+import cv2
+
+
+# using cpu
+ctx = mx.cpu(0)
+
+
+def parse_args():
+    """Training Options for Semantic Segmentation Experiments"""
```

Comment:
I think here is a typo, please rewrite it to depth demo.

Reply:
Fixed. Thanks for your reminder!